ci-operator/step-registry/cucushift/installer/wait/cucushift-installer-wait-commands.sh

```diff
@@ -1,4 +1,8 @@
 #!/bin/bash
 
-sleep "$SLEEP_DURATION" &
-wait
\ No newline at end of file
+if [ -f "${SHARED_DIR}/kubeconfig" ] ; then
```

Comment:
Pls do not change it. The step is referenced by many places. And the purpose of this step is for `wait`, does not care about kubeconfig file.